### DISCUSSION TITLE
naming the new command that adjusts config.txt, replacing "bootxxx" names

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#93] [Ideas]
In a discussion we were having , [appended to my issue here](https://github.com/gitbls/sdm/issues/85#issuecomment-1690742716), I suggested that a command that operates on the settings that rasp-config does be called --raspi-config. 

Here's a further suggestion: that the command that operates on and sets things in /boot/config.txt (your proposed replacement for bootadd and bootconfig) be called --config.txt.

Same idea, that this would instantly clue in experienced pi people to what the command effected, and it would give the new people a very likely-to-be-fruitful thing to google.

**Comment by gitbls** [comment]
After further thought, I like `raspiconfig` rather than `raspi-config`. Why? Having the same name creates confusion opportunities for new users because of the name collision. 

As far as config.txt, I REALLY like it, but that means there's a file named `plugins/config.txt`. Another potential source of confusion. So, I'm keeping this as bootconfig, as in `/boot/config`.

I may be over-rotating, but I have seen enough questions in all the various online forums, etc that lead me to believe that I'm not.

**Comment by tinker2much** [comment]
I like those: raspiconfig and bootconfig. They're distinct from each other, and they strongly suggest their roots.

**Comment by gitbls** [comment]
See https://github.com/gitbls/sdm/discussions/98
